code/modules/games/unum.dm

```diff
@@ -0,0 +1,33 @@
+/// A deck of unum cards. Classic.
+/obj/item/deck/unum
+	name = "\improper UNUM! deck"
+	desc = "A deck of unum cards. House rules to argue over not included."
```

Comment:
```suggestion
	desc = "A deck of UNUM! cards. House rules to argue over not included."
```
Consistent with the majority?

Reply:
Oops. I missed that one. I went around and changed it in all the other places.